client/my-sites/pages/main.jsx

```diff
@@ -100,12 +124,12 @@ class PagesMain extends React.Component {
 						fitsContainer
 						onSearch={ doSearch }
 						initialValue={ search }
-						placeholder={ this.props.translate( 'Search Pages' ) }
+						placeholder={ `${ searchPagesPlaceholder }...` }
```

Comment:
Can we use the ellipsis character rather than 2 dots?
```suggestion
						placeholder={ `${ searchPagesPlaceholder }…` }
```

Reply:
fixed